xapian-letor/tests/api_letor.cc

```diff
@@ -62,7 +87,7 @@ db_index_two_documents(Xapian::WritableDatabase& db, const string&)
     termgenerator.index_text("This is the paragraph of interest. Tigers are "
 			     "massive beasts - I wouldn't want to meet a "
 			     "hungry one anywhere. Lions are scary even when "
-			     "lyin' down. Bears are scary even when bare. "
+			     "lyin' down. Bears are scary even when bare."
```

Comment:
I suggested it would be better to fix the cases where a string literal is split and there was space both before and after the split.

But this case here (and the one below) aren't instances of that.

Once the compiler joins the adjacent string literals we now get "[...] Bears are scary even when bare.Together I suspect they'd be less scary, [...]" which is missing a space before "Together".

This change is not useful.